src/components/base/view.js

```diff
@@ -119,6 +119,9 @@ genotet.View.prototype.init = function() {
     .on('genotet.loadFail', function() {
       this.renderer.showFailure();
     }.bind(this));
+
+  // Override jQuery Resizable handle.
+  $('.ui-resizable-handle').css('z-index', 'auto');
```

Comment:
You are setting z-index for resizable-handles globally. However you should set z-index for handles only in this view.

Note that this method works only when the resizable-handles have a stable DOM order in the view. This is the case for now.